projects/angular/storyteller/src/app/routes/feed/feed.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="feed-page">
+  <mat-spinner [diameter]="70" *ngIf="isLoading"></mat-spinner>
+  <div class="align-container" *ngIf="!isLoading">
+    <div class="project-menu">
+      <project-select
+        [projectIdList]="projectIdList"
+        (selectEmitter)="loadProjectStories($event)"
+      ></project-select>
+    </div>
+    <div class="feed">
+      <app-story
```

Comment:
Here! A story is used in several different components. Perfect candidate to be shared.